web/src/constants/Filter.constants.ts

```diff
@@ -0,0 +1,6 @@
+import {Filter} from '../types/Common.types';
+
+export const filterList: Filter[] = [
```

Comment:
What is this for?

Reply:
I think we should move it away from the actual executable code and have a separated section for test utils, etc.